extensions/file/src/main/resources/META-INF/mule-module.properties

```diff
@@ -0,0 +1,8 @@
+module.name=file
+
+artifact.export.classPackages=org.mule.extension.file.api,\
```

Comment:
This shouldn't be done manually be the extensions user. Is there a jira already to make this automatically in ExtAPI?